content/news/033/index.md

```diff
@@ -72,6 +72,33 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Learning Material Updates
 
+### Game Development with Rust and WebAssembly
```

Comment:
Could you please add a ink to the book here?

Reply:
👍